newIDE/app/src/Utils/GDevelopServices/Authentication.js

```diff
@@ -59,24 +60,33 @@ export type RegisterForm = {|
   getNewsletterEmail: boolean,
 |};
 
-export type AdditionalUserInfoForm = {|
-  gdevelopUsage?: string,
-  teamOrCompanySize?: string,
-  companyName?: string,
-  creationExperience?: string,
-  creationGoal?: string,
-  hearFrom?: string,
-|};
-
 export type EditForm = {|
-  username: string,
-  description: string,
-  getGameStatsEmail: boolean,
-  getNewsletterEmail: boolean,
-  donateLink: string,
-  communityLinks: CommunityLinks,
+  +username: string,
+  +description: string,
+  +getGameStatsEmail: boolean,
+  +getNewsletterEmail: boolean,
+  +donateLink: string,
+  +communityLinks: CommunityLinks,
 |};
 
+export type PatchUserPayload = {
+  +username?: string,
+  +description?: string,
+  +getGameStatsEmail?: boolean,
+  +getNewsletterEmail?: boolean,
+  +appLanguage?: string,
+  +isCreator?: boolean,
+  +donateLink?: string,
+  +communityLinks?: CommunityLinks,
+  +survey?: UserSurvey,
+  +gdevelopUsage?: string,
+  +teamOrCompanySize?: string,
+  +companyName?: string,
+  +creationExperience?: string,
+  +creationGoal?: string,
+  +hearFrom?: string,
```

Comment:
are we still sending the last 6 props? (additionalUserInfo)

Reply:
Good idea to remove them!